### ISSUE TITLE
validate issue when using aws lambda connector

### ISSUE THREAD

**Issue by MartinDevaney** [llofberg/kafka-connect-aws-lambda#2]
Hi

I am having an issue with creating an instance of the AwsLambdaSinkConnector. I am using Landoop Connect UI on a non prod, 1 node cluster hosted on RHEL/Linux environment. Basically, as soon as I click to create an instance of the connector the Landoop UI doesn't display any of the properties to be defined and the 'Create' button is disabled. When I check the console I see that there is a 'validate' call to a url to the Connect REST api which is throwing an internal server error.

My understanding of what is happening is the default properties is sent for validation to Connect. Apparently the most recent version of Connect bombs if there is no topics or topics.regex in the properties. Not sure how to approach remedying this so reaching out to see if there is anything I can try. 
I am resorting to trying to use the 'confluent load' cli command but I am having problems getting it to parse the properties correctly (even though I am using a valid set from an existing connector). 

Any help would be greatly appreciated.
Thanks
Martin

Throws server error
http://<<ip withheld>>:8083/connector-plugins/com.tm.kafka.connect.aws.lambda.AwsLambdaSinkConnector/config/validate

Doesn't throw error
http://1<<ip withheld>>:8083/connector-plugins/io.confluent.connect.jdbc.JdbcSinkConnector/config/validate

![snip20180817_1](https://user-images.githubusercontent.com/39843582/44290274-d4cbdd80-a26f-11e8-8a03-9007a89e4b1a.png)

**Comment by llofberg** [comment]
Martin

I tried Landoop Connect UI with Kafka-connect-rest connector. It seems like deploying sink connectors doesn't work from the UI. I was able to deploy a source connector but sink failed even when the configuration validated against the validate REST call. 

Landoop seemed to suggest that both "topics" and "topics.regex" are required but then the REST validation fails with the following error.

"ConfigException: topics and topics.regex are mutually exclusive options, but both are set."

After adding the sink from command line with curl, Landoop did show it on the UI.

- lenny
